projects/tasks/AdapterTasks.wdl

```diff
@@ -598,6 +599,7 @@ task CopyToStagingBucket {
       declare -a DATA_FILES=(~{sep=' ' data_objects})
       declare -a REFERENCE_METADATA=(~{sep=' ' reference_metadata_objects})
       declare -a REFERENCE_FILE_DESCRIPTORS=(~{sep=' ' reference_file_descriptor_objects})
+      declare -a IS_UPDATE_FILE=(~{sep=' ' is_update_file})
```

Comment:
If you bring this in as a single string then you don't have to create this array
```suggestion
```

Reply:
so this was keeping with convention where we wrote all the files as arrays to the localized file and read from there...I can change the python for this single thing but i would prefer not to, let me know what you think

Reply:
would a middle ground be to make it clear with a comment that its always going to be a single element array

Reply:
oh wait a second, nvm you're totally right idk what im thinking, thanks!